g2p/constants.py

```diff
@@ -0,0 +1,43 @@
+"""
+This file is for constants that can be initialized without any (expensive) dependencies.
+"""
+import os
+from abc import ABC, abstractmethod
+
+DISTANCE_METRICS = [
+    "weighted_feature_edit_distance",
+    "hamming_feature_edit_distance",
+    "feature_edit_distance",
+    "dolgo_prime_distance",
+    "fast_levenshtein_distance",
+    "levenshtein_distance",
+]
+
+LANGS_DIR = os.path.join(os.path.dirname(__file__), "mappings", "langs")
+LANGS_FILE_NAME = "langs.json.gz"
+NETWORK_FILE_NAME = "network.json.gz"
+
+
+class BaseTransducer(ABC):
+    """Base class to typecheck transducers without having to import them."""
+
+    @abstractmethod
+    def __call__(self, to_convert: str):
+        pass
```

Comment:
You could add a `#pragma: no cover` here,  I suppose, or just a bogus test that calls this abstract method and ensures that it does nothing and returns `None`.

Likewise below (I'm not going to add another comment just to say the same thing)